RecoTracker/PixelSeeding/plugins/gpuPixelDoubletsAlgos.h

```diff
@@ -97,20 +119,26 @@ namespace gpuPixelDoublets {
 
   template <typename TrackerTraits>
   __device__ __forceinline__ void doubletsFromHisto(uint32_t nPairs,
+                                                    uint32_t maxNumOfDoublets,
                                                     GPUCACellT<TrackerTraits>* cells,
                                                     uint32_t* nCells,
                                                     CellNeighborsVector<TrackerTraits>* cellNeighbors,
                                                     CellTracksVector<TrackerTraits>* cellTracks,
                                                     HitsConstView<TrackerTraits> hh,
                                                     OuterHitOfCell<TrackerTraits> isOuterHitOfCell,
-                                                    CellCutsT<TrackerTraits> const& cuts) {
+                                                    CellCutsT<TrackerTraits> const* cuts) {
```

Comment:
why a `*` instead of a `&` ?
do we need to support the case where it's `nullptr` ?

Reply:
if it's never going to be `nullptr` (and it does not seem so), it would be better to keep it a `const&`.